README.md

```diff
@@ -54,7 +54,7 @@ This charm implements the following configs:
 And you can use it, like this:
 
 ```bash
-$  juju deploy ./mysql.charm --config MYSQL_ROOT_PASSWORD=SuperSecretPassword --config MYSQL_USER=JohnDoe --config MYSQL_PASSWORD=SuperSecretUserPassword --config MYSQL_DATABASE=default_database --resource mysql-image=ubuntu/mysql:lates
+$  juju deploy ./mysql-k8s.charm --config MYSQL_ROOT_PASSWORD=SuperSecretPassword --config MYSQL_USER=JohnDoe --config MYSQL_PASSWORD=SuperSecretUserPassword --config MYSQL_DATABASE=default_database --resource mysql-image=ubuntu/mysql:lates
```

Comment:
Shouldn't this be written as if deploying from charmhub?

Reply:
I was thinking in changing this once the charm is uploaded to charmhub